### DISCUSSION TITLE
USB-Ethernet driver

### DISCUSSION THREAD

**Discussion by kubaraczkowski** [Guilouz/Creality-Helper-Script-Wiki#533] [Features Request]
Hi,
What a wonderful project!

A general feature request from my side..
Would somebody be able to compile USB-Ethernet drivers (e.g. RTL8153 or AX88179 or other) for the MIPS linux platform and share the binaries?
This could be then included in the helper script!

Regards!

**Comment by peva3** [comment]
Bumping this up. The wifi on my k1c is essentially unusable and would love to get a more stable, higher throughput connection. This would be huge for people with print farms.

**Comment by NexGen-3D-Printing** [comment]
I know this is a more expensive option, but maybe we could swap out the control board with the K1 Max board, roughly $60 USD, I think they are identical boards, except for the ethernet port.

I might pick one up and try it, as I'm also not a fan of WiFi devices, specially 3D Printers, considering my printers are in ther same room as my server and switch rack, WiFi is just silly, my Max is awesome over Ethernet.
